lib/Sema/TypeCheckDeclOverride.cpp

```diff
@@ -591,6 +591,47 @@ static bool parameterTypesMatch(const ValueDecl *derivedDecl,
   return true;
 }
 
+static bool overridesDifferentiableAttribute(const ValueDecl *derivedDecl,
```

Comment:
Add a marker for the tensorflow branch.
```suggestion
// SWIFT_ENABLE_TENSORFLOW
static bool overridesDifferentiableAttribute(const ValueDecl *derivedDecl,
```